fdbserver/include/fdbserver/IPager.h

```diff
@@ -78,11 +80,7 @@ static const std::vector<std::pair<PagerEvents, PagerEventReasons>> L0PossibleEv
 	{ PagerEvents::PageWrite, PagerEventReasons::MetaData },
 };
 
-enum EncodingType : uint8_t {
-	XXHash64 = 0,
-	// For testing purposes
-	XOREncryption = 1
-};
+enum EncodingType : uint8_t { XXHash64 = 0, XOREncryption_TestOnly = 1, Encryption = 2, MAX_ENCODING_TYPE = 3 };
```

Comment:
Since we may add other encoding types in the future, we probably should call `Encryption` something more specific to what the scheme it represents is.

Reply:
I think for now this encoding type is the only encryption that we have. And I think it is generic enough, for example, the encoding doesn't have anything to do with tenant and the `RandomEncryptionKeyProvider` for the unit test use the same encoding. On the other hand, it has a bunch of implementation details (that it has additional SHA256 digest, and the digest is using a separate cipher key, for example) that it is hard to capture in a good name. I think I can change it to `AESEncryption` to be able to tell it apart from `XOREncryption` though. Let me know if it looks better.

Reply:
Yeah I was thinking something like `AESEncryptionV1` just to sort of describe the scheme the the encoder uses.  Leaving it as `Encryption` makes it seem like under this mode other algorithms could be used but that wouldn't be practical.